### PR TITLE
remove excess memberships; make Jesse only GA owner; fixes #138

### LINKED ISSUE #138: Figure out a way to get rid of the gazillion excess memberships

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

**Comment by RobertAKARobin** [comment]
Tested with old production data.

File: db/migrate/20151104040728_remove_excess_memberships.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class RemoveExcessMemberships < ActiveRecord::Migration
+  def change
+    Group.at_path("ga").memberships.where(is_owner: true).destroy_all
+    Group.at_path("ga_wdi").memberships.destroy_all
+    Group.at_path("ga_wdi_dc").memberships.destroy_all
+
+    User.named("garoot").destroy
+    Group.at_path("ga").add_owner("robertakarobin")
+  end
```
Comment (mattscilipoti) [inline db/migrate/20151104040728_remove_excess_memberships.rb]: Maybe an instructor and Brian?

**Comment by mattscilipoti** [comment]
:+1: Recommend adding Brian (and/or Jon) as ga owner

File: db/migrate/20151104040728_remove_excess_memberships.rb (hunk 1 above)
Comment (mattscilipoti) [inline db/migrate/20151104040728_remove_excess_memberships.rb]: Recommend using `delete` in migrations, unless there is a conscious choice to run all events and dependency constraints (dependent :destroy).

**Comment by mattscilipoti** [comment]
Added note about delete vs. destroy.

**Comment by RobertAKARobin** [comment]
@mattscilipoti Thanks! Fixed according to your suggestions.

**Comment by jshawl** [comment]
:+1: This one makes me a little nervous. Do we have a backup @adambray ?

**Comment by RobertAKARobin** [comment]
`pg_dump` ftw!

**Comment by RobertAKARobin** [comment]
@jshawl Changed this to you

**Pull request merged**